### ISSUE TITLE
a shape background for each character

### ISSUE THREAD

**Issue by GSven** [robinhood/ticker#52]
can add a shape or image background for each character ?

**Comment by jinatonic** [comment]
Yeah something we currently don't support is having custom draw logic for each character. I'm going to use this issue to track that work, but not sure when I can get to it. Unfortunately there is no current way of doing what you want unless you want to fork the repo and make the customizations yourself.

**Comment by danh32** [comment]
Not sure about your exact use-case, but customizing each character sounds like a custom font. If you rolled your own font, you could just apply that typeface to Ticker and that would work.

**Comment by lcyluo** [comment]
And hao can i draw a background drawable just use shape or res to each character？
